docker/script/bootstrap.sh

```diff
@@ -34,14 +34,13 @@ tar -xf ./python_install/$python_tar -C ./python_install
 dnf install -y dnf-plugins-core
 dnf builddep -y python3
 
-
 pushd /python_install/$python_file 
 ./configure 
 make install -j $(nproc) # use -j to set the cores for the build
 popd
 
 # Upgrade pip
-pip3 install $PIP_OPTION --upgrade 'pip<23'
+pip3 install $PIP_OPTION --upgrade 'pip<23' cryptography
```

Comment:
Minor: This can be a completely new  line like this:

# Needed for generating fernet key for local runner
pip3 install cryptography --constraint /constraints.txt